integration/network/service_test.go

```diff
@@ -204,8 +214,7 @@ func TestDaemonWithBipAndDefaultNetworkPool(t *testing.T) {
 	out, err := c.NetworkInspect(context.Background(), "bridge", types.NetworkInspectOptions{})
 	assert.NilError(t, err)
 	// Make sure BIP IP doesn't get override with new default address pool .
-	assert.Equal(t, out.IPAM.Config[0].Subnet, "172.60.0.0/16")
-	delInterface(t, defaultNetworkBridge)
+	assert.Equal(t, out.IPAM.Config[0].Subnet, "172.60.0.1/16")
```

Comment:
Note to self (I'm rebasing this one); check why I changed this from `"172.60.0.0/16"` to `"172.60.0.1/16"`